README.md

```diff
@@ -1,8 +1,5 @@
 # Amphora-Atom
 
-[![CircleCI](https://circleci.com/gh/clay/amphora-atom/tree/master.svg?style=svg)](https://circleci.com/gh/clay/amphora-atom/tree/master)
```

Comment:
I think we can leave the CircleCI Badge for now just to know if the build is passing or not